widgetssdk/src/main/java/com/glia/widgets/call/CallView.kt

```diff
@@ -304,8 +304,7 @@ internal class CallView(
             operatorStatusView.showTransferring()
             operatorNameView.text = (stringProvider.getRemoteString(R.string.engagement_queue_transferring_message))
         } else {
-            operatorNameView.text = stringProvider.getRemoteString(R.string.general_company_name)
-            operatorNameView.hint = stringProvider.getRemoteString(R.string.chat_operator_name_accessibility_label)
```

Comment:
What about accessibility?

Reply:
@Pelkar It seems to me that we need a separate string that will be used as an accessibility label. Basically, it should be a translation for the "Company name".

So, one string for the name itself and another one just as a translation for "Company name".

As far as I understand, currently, we have only one string for the name.

Reply:
Or I can just use the name of the company as an accessibility label.

Reply:
@AndriiHorishniiMOC Fixed